product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/18_dbms_xmldom/index.mdx

```diff
@@ -20,26 +20,10 @@ navigation:
 - setversion
 ---
 
-The `DBMS_XMLDOM` package is mainly used to access the `XMLType` objects. It provides an application programming interface for HTML and XML documents as an Document Object Model (DOM). This package is specially designed to work like any other DOM interfaces.
+The `DBMS_XMLDOM` package implements Document Object Model (DOM) and is used to create DOM documents from scratch or from an `XMLType` object.
```

Comment:
I apologise for attempting to correct this again.
The `DBMS_XMLDOM` package implements Document Object Model (DOM) and is used to create DOM documents from scratch or from an XML document.